en/docs/debugging/debug.md

```diff
@@ -0,0 +1,258 @@
+# Debug Ballerina Programs
+
+When writing large-scale applications with complex logic, it is quite possible to have syntax, semantic,
```

Comment:
```suggestion
When writing large-scale applications with complex logic, you may encounter syntax, semantic,
```

Reply:
+1

Reply:
added with [Address the latest review suggestions](https://github.com/wso2/docs-ballerina-visual-studio-code-plugin/pull/20/commits/71d66faeecce06a07d7eccec8edf1b3c3eb42ffb)